pgo/cmd/auth.go

```diff
@@ -155,7 +159,7 @@ func GetCredentialsFromFile() msgs.BasicAuthCredentials {
 	return creds
 }
 
-func GetCredentialsFromEnvironment() msgs.BasicAuthCredentials {
+func getCredentialsFromEnvironment() msgs.BasicAuthCredentials {
```

Comment:
Great change! Please document this function if we're going to be renaming it ;-)

Reply:
Function is now documented.